raiden/tests/integration/network/transport/rif_comms/test_client.py

```diff
@@ -23,92 +24,173 @@ def comms_nodes(amount_of_nodes) -> {int, CommsNode}:
 
 
 @pytest.mark.parametrize("amount_of_nodes", [1])
-def test_locate_own_peer_id(comms_nodes):
-    comms_node = comms_nodes[1]
+def test_subscribe_to_invalid(comms_nodes):
+    client = comms_nodes[1].client
+    # subscribe to unregistered address
+    topic_id, _ = client.subscribe_to(generate_address())
 
-    # attempt to locate self
-    assert comms_node.client._get_peer_id(comms_node.address) is not ""
+    # FIXME: this should probably throw an exception rather than an empty result
```

Comment:
This is really important, since we are going to threat each incorrect behaviour as an exception. 

The subscribe to flow into the payment of lumino, should check for SubscriberError

Lets report this

Reply:
> This is really important, since we are going to threat each incorrect behaviour as an exception.
> 
> The subscribe to flow into the payment of lumino, should check for SubscriberError
> 
> Lets report this

it's in the pending issues doc, i'm in the process of describing it in full